modules/rosa-edit-objects.html

```diff
@@ -0,0 +1,761 @@
+<!DOCTYPE html>
```

Comment:
Is there a reason this is an html file? Its not coming up in the preview from what I can see.

Reply:
Thank you - I'm not sure what is up with this, but it looks okay I think. I don't see it in the preview either.